<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encryption Methods</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EncryptFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AES_Radio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="DES_Radio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendFileButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send File</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="BLOWFISH_Radio">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BLOWFISH</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputKey">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Local Video Stream</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GenerateKeyButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Software Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypted Data</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="EncryptedText">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameInput">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PingButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>640</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ping Users</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="LogsView">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>371</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DecryptFileButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>430</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReceiveFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File To Receive</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewFileInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>New File Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="LocalVideoStream">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>encryption.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SendStream">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start  Encrypted Stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EndStream">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>End Stream</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopLocalStream">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartLocalStream">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadFPGA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Read From External Hardware</string>
    </property>
   </widget>
   <widget class="QComboBox" name="PortsCombo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="HardWareStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hardware Connection Status ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hardware Communication Settings</string>
    </property>
   </widget>
   <widget class="QComboBox" name="BaudRateCombo">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string>128000</string>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>55</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BaudRate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendKeyButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Key Through RSA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="KeySendUser">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1299</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
